src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,127 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        class PersonAgeComparator implements Comparator<Person> {
+
+            @Override
+            public int compare(Person firstPerson, Person secondPerson) {
+                return firstPerson.getAge() - secondPerson.getAge();
+            }
+        }
+
+        Arrays.sort(persons, new PersonAgeComparator());
+
+        assertArrayEquals(new Person[]{
+            new Person("Иван", "Мельников", 20),
+            new Person("Николай", "Зимов", 30),
+            new Person("Алексей", "Доренко", 40),
+            new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person firstPerson, Person secondPerson) {
+                return firstPerson.getAge() - secondPerson.getAge();
```

Comment:
It's better to use ```Integer.compare(..., ...)``` because of potential overflow. 
See: https://stackoverflow.com/a/2728810

Reply:
Fixed here: a34350314417f029e18bf78ce0090cc8b4e11b1b